code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -516,6 +522,35 @@
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/toggle_biolum,
```

Comment:
Yeah, don't do this, please. Verbs are gross and unwieldy to use; this should be an action button handled on the organ itself.